### PR TITLE
Remove 7 zh files that not exist in en upstream currently

### PR DESCRIPTION
**Pull request by liu-xg**
doc sync

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
Not sure it's applicable or not to simply remove those files. The CI tells failed now:

```
1:54:21 PM: ERROR Could not find glossary entry for 'Mixer' (/opt/build/repo/content/zh/blog/2019/performance-best-practices/index.md:35:67)
```

**Comment by wilsonwu** [comment]
Maybe some case study docs only for Chinese version? Need more background from other maintainers.

File: content/zh/blog/2021/istiomeetups-china/index.md
Comment (wilsonwu) [inline content/zh/blog/2021/istiomeetups-china/index.md:1]: I think this should not be removed.

Comment (liu-xg) [reply content/zh/blog/2021/istiomeetups-china/index.md:1]: ok

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/zh/about/case-studies/redhat/index.md
Comment (wilsonwu) [inline content/zh/about/case-studies/redhat/index.md:1]: Anyone can help to comfirm the redhat and vmware case study is only for Chinese version or need to be removed?

**Comment by windsonsea** [comment]
We need to check the history of those en upstream files one by one and find why and when. That is why not suitable to include many breaking changes in a PR.

**Comment by wilsonwu** [comment]
> We need to check the history of those en upstream files or commit history one by one and find why and when. That is why not suitable to include many breaking changes in a PR.

Checked the closed PRs with keywords, but no useful info, it is a huge work to check code base.

File: content/zh/about/case-studies/redhat/index.md
Comment (hanxiaop) [reply content/zh/about/case-studies/redhat/index.md:1]: Looks like these things were removed by @craigbox  in https://github.com/istio/istio.io/pull/9681

Comment (hanxiaop) [reply content/zh/about/case-studies/redhat/index.md:1]: I think the files can be removed but maybe Craig knows these things better.

Comment (craigbox) [reply content/zh/about/case-studies/redhat/index.md:1]: those were only design placeholders; they were never meant to be live.

**Review by windsonsea — APPROVED** [review]
Thanks all.

**Pull request merged**